src/execution/parsing/dag_parser.c

```diff
@@ -69,7 +69,7 @@ static int _ParseDAGLoadArgs(RedisModuleCtx *ctx, RedisModuleString **argv, int
 
     if (number_loaded_keys != n_keys) {
         RAI_SetError(err, RAI_EDAGBUILDER,
-                     "ERR number of keys to LOAD in AI.DAGRUN command does not match the number of "
+                     "ERR number of keys to LOAD into DAG does not match the number of "
```

Comment:
The previous phrasing is more informative

Reply:
Same as above...